### ISSUE TITLE
[1.12.2] "Lava Block" config doesn't work correctly

### ISSUE THREAD

**Issue by aks746** [yungnickyoung/Better-Caves#106]
Hello! 

First of all sorry for the bad English.

I am trying to replace the lava in the overworld caves with a different type of liquid, but the flowing lava from the pockets still appears in the caves. I tested this without other mods and turned off all caves except floored caverns/liquid caverns and tried to use this option:

>             # The block used for lava generation at and below the Liquid Altitude.
>             #     Defaults to regular lava if an invalid block is given.
>             # Default: minecraft:lava
>             S:"Lava Block"=minecraft:water

Underground lava lakes are replaced with water correctly, but as I said, the flowing lava from the pockets on the walls still appears.
Is there some way to fix this?

Minecraft 1.12.2
Forge 14.23.5.2854
Bettercaves-1.12.2-2.0.4

P.S. And many thanks for the cool mod by the way!

**Comment by Lughbaloo** [comment]
That's a decoration feature in base minecraft. Unfortunately, Better Caves only focuses on cave generation, and doesn't handle the features inside the caves. I personally don't know of a mod that changes this, but you might even be able to remove it with a datapack.

The "Lava Block" feature doesn't mean its blocking out lava, it means what minecraft block is being used instead of the lava block (in this case its a fluid). So from what you've mentioned, it's working correctly.

**Comment by aks746** [comment]
Ok, I got it!

Eventually I added a short script using [BiomeTweaker](https://www.curseforge.com/minecraft/mc-mods/biometweaker). If someone comes across a similar problem for 1.12.2, then something like this can be used:

```
Lava = forAllBiomes()
Lava.removeDecoration("LAKE_LAVA")
Lava.removeFeature("LAVA")
```

However, I think it will be great if the option to completely replace the lava appears in the Better Caves. This would be very helpful for accurate customization.
